Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Use a base image with JDK and Gradle
+FROM openjdk:17-jdk-slim
+
+# Set the working directory for the runtime environment
+WORKDIR /app
+
+# Copy the compiled JAR file from the build stage
+COPY ./build/libs/delivery-0.0.1-SNAPSHOT.jar app.jar
```

Comment:
you can use `delivery-*.jar` instead of specifying the version

Reply:
why is it resolved without any reply or changes?